apps/agentstack-cli/src/agentstack_cli/commands/server.py

```diff
@@ -135,18 +135,24 @@ async def server_login(server: typing.Annotated[str | None, typer.Argument()] =
         auth_server = None
         token = None
 
-        client_id = config.client_id
+        client_id = None
         client_secret = config.client_secret
 
         if auth_servers:
             if len(auth_servers) == 1:
-                auth_server = auth_servers[0]
+                auth_server = auth_servers[0].get("server", "")
+                client_id = auth_servers[0].get("client_id", "")
```

Comment:
`auth_servers` is a list of strings as defined in https://datatracker.ietf.org/doc/html/rfc9728, it is not an object containing `server` and `client_id` fields.